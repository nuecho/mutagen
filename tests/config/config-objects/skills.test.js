cfgObjectTests("skills", "config-objects/skills-config.json");
