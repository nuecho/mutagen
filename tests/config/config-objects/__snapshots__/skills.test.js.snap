// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen skills config import-export] should create the skills: the skills have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ skill [Environment/cfgObjectTest]
Completed. 1 object created. 0 object updated. 0 object skipped.
",
}
`;

exports[`[mutagen skills config import-export] should skip the same skills: the skills have not been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "= skill [Environment/cfgObjectTest]
Completed. 0 object created. 0 object updated. 1 object skipped.
",
}
`;

exports[`[mutagen skills config import-export] should update the modified skills: the modified skills have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ skill [Environment/cfgObjectTest]
Completed. 0 object created. 1 object updated. 0 object skipped.
",
}
`;
